userguide/snippets/typography.rst

```diff
@@ -34,3 +34,11 @@
    +----------------------------------------------------------------+---------------------------------------------------+
    | *Launch* icon                                                  | Click |ui-launch| to open the netdata interface.  |
    +----------------------------------------------------------------+---------------------------------------------------+
+   | *Play* icon                                                    | |ui-jail-start| starts the selected jails.        |
```

Comment:
s/Play/Start/, but wouldn't this table be clearer without repeating the verb?  That is, show me, don't tell me.  "The selected jails" is not needed, it was explained above.  Also, not "Starts", but just "Start".  (Not future tense?  I can't think what to call this right now.)

| |ui-jail-start|        | Start the jail. |